test/test_parse.js

```diff
@@ -225,6 +225,12 @@ describe('SMD Resistors', () => {
     assert(c.resistance === 1, 'resistance value is wrong')
     assert(c.size === '0402', 'size is wrong')
   })
+  it('parses "1k ohms"', () => {
+    const c = parse('1k ohm 0603')
```

Comment:
I think you forgot to change the test here.

Reply:
Ouch. Thanks for pointing it out! Fixed it.